modules/indexer/issues/indexer.go

```diff
@@ -310,6 +323,7 @@ func DeleteRepoIssueIndexer(repo *models.Repository) {
 }
 
 // SearchIssuesByKeyword search issue ids by keywords and repo id
+// WARNNING: You have to ensure user have permission to visit repoIDs' issues
```

Comment:
This messes up with our paging, but I don't see much way around it.

Reply:
Yes. I left it for future PR.